### DISCUSSION TITLE
Is it possible to provide input when Creating a new note from template?

### DISCUSSION THREAD

**Discussion by CodyBontecou** [SilentVoid13/Templater#719] [Help] [answered]
Like the title asks, is it possible to be given an input field when creating a new note from a template?

I want to set a variable, such as the note title, while creating the new note that will be used as a variable throughout the new note.

For example, I select `Templater: Create new note from the template.`

I then want an input form to appear asking what I would like the note's title to be. Let's say I input `Random Blog Post`.

This then generates a note with metadata autogenerated based on the inputted data like:

    ---
	title: 'Random Blog Post'
	slug: 'random-blog-post'
        canonical_link: 'https://codybontecou.com/random-blog-post'
        meta:
                - name: og:title
                   content: My Blog | Cody Bontecou
                - name: og:image
                   content: https://codybontecou.com/images/random-blog-post-meta.png
    ---

**Comment by AB1908** [comment, marked as answer]
You can use `tp.system.prompt`. Check docs and also browse forum and GitHub discussions for examples.

**Reply by CodyBontecou** [reply]
Oo yup, that's perfect. Thank you.

**Reply by AB1908** [reply]
This isn't necessary but I'd appreciate you changing this discussion into a question and accepting my answer. Nothing useful or practical, I just like the green dot on my contributions lol.

**Reply by CodyBontecou** [reply]
No worries. I think I did this. Lmk if you need anything else.
